src/main/java/pl/wojciechkostecki/revel/model/Local.java

```diff
@@ -21,5 +21,7 @@
 
     private LocalTime closingTime;
 
-    private String menu;
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "menu_id", referencedColumnName = "id")
+    private Menu menu;
```

Comment:
This entity actually reminded me that I have actually written some of the code you are trying to write year or two ago when our name was still PubAid. I actually copied these repo's from AWS to github couple days ago so i can give you access to them if you want to have a look at the code. I don't remember if it was written in Java though :)

Reply:
Sure I'd like to see this code.

Reply:
I just checked and I have lost that code :(. I think I didn't copy all of the code to github